### ISSUE TITLE
Improve faster loading time on listing roms

### ISSUE THREAD

**Issue by animekai** [Rinnegatamante/DaedalusX64-vitaGL#127]
There is any way to improve the speed of listing games? i got the fullromset, 7gb of games, and takes too long to list the games, ive also disabled the autoupdate feature to improve speed but many games take as 15 or 20 second to list.

maybe it could be a one time scan or list refresh to improve speed?

**Comment by Rinnegatamante** [comment]
This is most likeely caused by the rom sorting code if someone wants to pick this.

**Comment by jona82** [comment]
good if it takes a long time to load the roms for those who have a big list of games, you could create or choose a list of roms and make them as favorites so they are faster to load.

**Comment by animekai** [comment]
Or maybe with a cache list with a manual or automatic scan, or allow to scan subfolders to put files in folders like from a to z folders.

**Comment by Rinnegatamante** [comment]
Is this still a thing?

**Comment by animekai** [comment]
yes, when having lots of roms (500) takes around 20 or more second to load daedalus into rom list, when having 100 or less it takes5 secods or less

with autoupdates disabled obviously
